plugin/trino-hive/src/main/java/io/trino/plugin/hive/orc/OrcReaderConfig.java

```diff
@@ -152,4 +152,19 @@ public OrcReaderConfig setNestedLazy(boolean nestedLazy)
         options = options.withNestedLazy(nestedLazy);
         return this;
     }
+
+    @Deprecated
+    public boolean isShortZoneIdEnabled()
+    {
+        return options.isShortZoneIdEnabled();
+    }
+
+    @Deprecated
+    @Config("hive.orc.short-zone-id-enabled")
+    @ConfigDescription("ORC read stripe footer with short zone id")
```

Comment:
Can we reword this description to something like `Allow reads on ORC with short zone id in footer`